generator.yaml

```diff
@@ -485,6 +485,38 @@ resources:
         from:
           operation: DescribeMonitoringSchedule
           path: LastMonitoringExecutionSummary
+  ModelPackageGroup:
+    exceptions:
+      errors:
+          404:
+            code: ValidationException
+            message_suffix: does not exist.
+      terminal_codes:
+        - ResourceLimitExceeded
+        - ResourceNotFound
+        - ResourceInUse
+        - OptInRequired
+        - InvalidParameterCombination
+        - InvalidParameterValue
+        - MissingParameter
+        - MissingAction
+        - InvalidQueryParameter
+        - MalformedQueryString
+        - InvalidAction
+        - UnrecognizedClientException
+    fields:
+      ModelPackageGroupStatus:
+        is_read_only: true
+        print:
+          name: STATUS
+        from:
+          operation: DescribeModelPackageGroup
+          path: ModelPackageGroupStatus
+      CreationTime:
```

Comment:
not really useful unless there is `lastModifiedTime` as well

Reply:
There is not for this resource :(

Reply:
ok, let's remove it then, created = ARN exists